### PR TITLE
devise setup

### PR DESCRIPTION
**Pull request by ropergroup**
Hello! This is the devise setup with a little start to the registered_application CRUD.

### CODE REVIEW

**Comment by brittanyjspringer** [comment]
@roperxyz now that you are one project in, it is time for me to be more picky. I'm going to tag some files you need to delete since they are empty. If you commit a file that you are not going to use, get in the habit of removing it.

File: app/assets/javascripts/registered_applications.coffee [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# Place all the behaviors and hooks related to the matching controller here.
```
Comment (brittanyjspringer) [inline app/assets/javascripts/registered_applications.coffee]: Delete please.

File: app/assets/stylesheets/registered_applications.scss [hunk 2]
```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the RegisteredApplications controller here.
```
Comment (brittanyjspringer) [inline app/assets/stylesheets/registered_applications.scss]: Delete please.

File: app/assets/stylesheets/registered_applications_controller.scss [hunk 3]
```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the RegisteredApplicationsController controller here.
```
Comment (brittanyjspringer) [inline app/assets/stylesheets/registered_applications_controller.scss]: Delete please.

File: app/helpers/registered_applications_controller_helper.rb [hunk 4]
```diff
@@ -0,0 +1,2 @@
+module RegisteredApplicationsControllerHelper
```
Comment (brittanyjspringer) [inline app/helpers/registered_applications_controller_helper.rb]: Delete please.

File: app/helpers/registered_applications_helper.rb [hunk 5]
```diff
@@ -0,0 +1,2 @@
+module RegisteredApplicationsHelper
```
Comment (brittanyjspringer) [inline app/helpers/registered_applications_helper.rb]: Delete please.

File: config/initializers/devise.rb [hunk 6]
```diff
@@ -0,0 +1,274 @@
+# Use this hook to configure devise mailer, warden hooks and so forth.
+# Many of these configuration options can be set straight in your model.
+Devise.setup do |config|
+  # The secret key used by Devise. Devise uses this key to generate
+  # random tokens. Changing this key will render invalid all existing
+  # confirmation, reset password and unlock tokens in the database.
+  # Devise will use the `secret_key_base` as its `secret_key`
+  # by default. You can change it below and use your own secret key.
+  # config.secret_key = '2bb852174050537e5bcd99f53f590c5132854e4fb6ab4666e1fb6eb12340fe60bf6172a4b565f9594a9091a120b6ff313484f16dec4d1913f754301b6a1138c1'
```
Comment (brittanyjspringer) [inline config/initializers/devise.rb:9]: Don't forget to uncomment.

File: spec/helpers/registered_applications_controller_helper_spec.rb [hunk 7]
```diff
@@ -0,0 +1,15 @@
+require 'rails_helper'
```
Comment (brittanyjspringer) [inline spec/helpers/registered_applications_controller_helper_spec.rb]: Delete please.

File: spec/helpers/registered_applications_helper_spec.rb [hunk 8]
```diff
@@ -0,0 +1,15 @@
+require 'rails_helper'
```
Comment (brittanyjspringer) [inline spec/helpers/registered_applications_helper_spec.rb]: Delete please.

**Comment by brittanyjspringer** [comment]
Also, why does it look like you skipped right into Registered Applications? Please try to do the Checkpoints in order so I can follow along.

File: config/initializers/devise.rb (hunk 6 above)
Comment (ropergroup) [inline config/initializers/devise.rb:9]: We need to talk about this...

**Comment by ropergroup** [comment]
The first three checkpoints all have to do with devise. So I thought I would complete those checkpoint and send it to you in one lump. I do have a few issues that I have listed on the google doc.

**Comment by ropergroup** [comment]
I am running into an issue. I have continued forward so I have made the changes you asked for but I also have the current work I have be working on. I am not sure how to keep the two separated.

**Comment by brittanyjspringer** [comment]
@roperxyz you can pull down your Master branch and merge into your working branch. Make sure your features are always on separate branches.

**Pull request merged**